src/lib.rs

```diff
@@ -41,6 +39,6 @@ extern {
     pub fn tinyfd_colorChooser (
         aTitle: *const c_char,
         aDefaultHexRGB: *const c_char,
-        aDefaultRGB: [c_uchar ; 3],
-        aoResultRGB: [c_uchar; 3]) -> *const c_char;
+        aDefaultRGB: &[c_uchar ; 3],
+        aoResultRGB: &[c_uchar; 3]) -> *const c_char;
```

Comment:
Turns out I was wrong and this isn't safe either (since in Rust `&[T; N]` is represented as `(*const T, usize)`). This will need to be `*const c_uchar` instead.